shared/storybook/index.native.js

```diff
@@ -34,23 +35,85 @@ const loadStories = () => {
   }, module)
 }
 
-const rootDecorator = story => (
-  <Kb.Box style={styles.container}>
-    <KeyboardAvoidingView
-      behavior={Styles.isIOS ? 'padding' : undefined}
-      enabled={true}
-      style={styles.keyboard}
-    >
-      <Kb.Box style={styles.storyWrapper}>
-        <StatusBar key="statusbar" hidden={true} />
-        {story()}
-      </Kb.Box>
-    </KeyboardAvoidingView>
-  </Kb.Box>
-)
+function useInterval(callback, delay) {
+  const savedCallback = React.useRef()
+
+  React.useEffect(() => {
+    savedCallback.current = callback
+  }, [callback])
+
+  React.useEffect(() => {
+    function tick() {
+      const c = savedCallback.current
+      c && c()
+    }
+    if (delay !== null) {
+      let id = setInterval(tick, delay)
+      return () => clearInterval(id)
+    } else return undefined
+  }, [delay])
+}
+// keep modes in the module so its kept between stories
```

Comment:
You mean so if you go to another store and come back, this is saved?

Reply:
yes, when you switch stories it remounts the whole thing. this way if you switch it keeps the mode. same things happens on desktop